src/native-stack/navigators/createNativeStackNavigator.tsx

```diff
@@ -35,6 +35,18 @@ function NativeStackNavigator({
     screenOptions,
   });
 
+  // Starting from React Navigation v6 @react-navigation/native-stack should be used.
+  // react-native-screens/native-stack works kinda okay for v6 (still) but types
+  // are different and it isn't supported for v6 so we have to warn users about this.
```

Comment:
Is this information really needed? I am not sure if it will always work like that, maybe there will be some changes that will break compatibility between those packages.

Reply:
Fixed in 7c22e558423fd3d66c242c23092e9288c8cf69a9